test/e2e/v1beta1/scripts/gh-actions/build-load.sh

```diff
@@ -34,6 +34,7 @@ TAG="e2e-test"
 VERSION="v1beta1"
 CMD_PREFIX="cmd"
 SPECIFIED_DEVICE_TYPE_IMAGES=("enas-cnn-cifar10-cpu" "darts-cnn-cifar10-cpu" "pytorch-mnist-cpu")
+DEFAULT_IMAGE_FOR_TUNE="docker.io/tensorflow/tensorflow:2.13.0"
```

Comment:
We don't need this since we use this image as a default here: https://github.com/kubeflow/katib/blob/master/sdk/python/v1beta1/kubeflow/katib/api/katib_client.py#L169

Reply:
I encountered some errors when not pulling this image. But I can have try to test it again to confirm this.